mne/io/kit/tests/test_kit.py

```diff
@@ -185,4 +188,24 @@ def test_hsp_elp():
     assert_array_almost_equal(pts_elp_in_dev, pts_txt_in_dev, decimal=5)
 
 
+def test_decimate():
+    """Test decimation of digitizer headshapes with too many points. """
+    # load headshape and convert to meters
+    hsp = np.loadtxt(sphere_hsp_txt_path)
+    hsp /= 1000
```

Comment:
Do not add 700+kB for a sphere of points. We can generate them easily. Is it slow or something? If so just do a `meshgrid` with azimuth/polar and `sph_to_cart` and it will be fast.